Schema.js

```diff
@@ -75,6 +78,7 @@ function classNameIsValid(className) {
     className === '_Session' ||
     className === '_SCHEMA' || //TODO: remove this, as _SCHEMA is not a valid class name for storing Parse Objects.
     className === '_Role' ||
+    className === '_GlobalConfig' ||
```

Comment:
That will work but then we need one for `update` too because the current `update` also uses `database.collection`